calico_kubernetes/calico_kubernetes.py

```diff
@@ -726,15 +728,55 @@ def run_protected():
     """
     Runs the plugin, intercepting all exceptions.
     """
-    # Configure logging
-    configure_logger(logger, LOG_LEVEL, True)
-    configure_logger(pycalico_logger, LOG_LEVEL, False)
+    # Parse arguments and configure logging
+    global logger
+    mode = sys.argv[1]
+
+    # Append a stdout logging handler to log to the Kubelet.
+    # We cannot do this in the status hook because the Kubelet looks to
+    # stdout for status results.
+    if mode != 'status':
+        configure_stdout_logger(logger=logger,
```

Comment:
Can we get round this by just making all our logs to stderr? I think this fix makes the logic here a bit tough to follow.